src/main/java/net/dv8tion/jda/api/events/user/update/UserUpdateAvatarEvent.java

```diff
@@ -71,6 +72,22 @@ public String getOldAvatarUrl()
         return previous == null ? null : String.format(User.AVATAR_URL, getUser().getId(), previous, previous.startsWith("a_") ? "gif" : "png");
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this user's old avatar image
+     * <p>
+     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this user's old avatar image.
     * <p>
     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord.
```